sites/laserfocusworld/server/templates/index.marko

```diff
@@ -61,12 +61,13 @@ $ const adSlots = {
         </div>
         <div class="col-lg-6 mb-block">
           <endeavor-content-query-section-list
-            limit=3
-            section-alias="software-accessories"
-            header={ title: "Software Accessories", href: "/software-accessories" }
-            native-x={ placement: 'list1', aliases: ['software-accessories'], index: 2 }
+            limit=4
```

Comment:
This should be set to `limit=3` to match the adjacent block limit.
The `skip=5` should also be removed.

Reply:
commit a44df94